scala_proto/scala_proto.bzl

```diff
@@ -319,38 +319,39 @@ def _colon_paths(data):
     return ':'.join(["{root},{path}".format(root=_root_path(f), path=f.path) for f in data])
 
 def _gen_proto_srcjar_impl(ctx):
-    acc_imports = depset()
+    acc_imports = []
     transitive_proto_paths = depset()
 
     proto_deps, jvm_deps = [], []
     for target in ctx.attr.deps:
         if hasattr(target, 'proto'):
             proto_deps.append(target)
-            acc_imports += target.proto.transitive_sources
+            acc_imports.append(target.proto.transitive_sources)
             #inline this if after 0.12.0 is the oldest supported version
             if hasattr(target.proto, 'transitive_proto_path'):
               transitive_proto_paths += target.proto.transitive_proto_path
         else:
             jvm_deps.append(target)
 
+    acc_imports = depset(transitive = acc_imports)
```

Comment:
Can we move changing acc_imports to a depset after 44? 
In it you’re changing it back to a list which seems redundant

Reply:
so, note that `acc_imports` as the arg to transitive is a `List[Depset]` not a `List[File]` or something, so when we do `depset(transitive = acc_imports).to_list()` converts a `List[Depset]` to a `List[File]`.

Reply:
😱 that's so implicit JavaConversions feels explicit 😉